app/src/sharedTest/java/org/oppia/android/app/databinding/StateAssemblerPaddingBindingAdaptersTest.kt

```diff
@@ -147,10 +147,10 @@ class StateAssemblerPaddingBindingAdaptersTest {
       )
       return@runWithActivity textView
     }
-    /**
-     * Note that the padding starts/ends below match the ones set above because, when the adapters are
-     * working correctly in RTL mode, the start/end should be exactly the start/end originally set.
-     */
+    /*
+    * Note that the padding starts/ends below match the ones set above because, when the adapters are
+    * working correctly in RTL mode, the start/end should be exactly the start/end originally set.
+    */
```

Comment:
```suggestion
    /*
     * Note that the padding starts/ends below match the ones set above because, when the adapters are
     * working correctly in RTL mode, the start/end should be exactly the start/end originally set.
     */
```

Nit on formatting--please fix elsewhere.